src/client/components/line-chart-axis/line-chart-axis.tsx

```diff
@@ -36,37 +36,42 @@ export interface LineChartAxisProps {
   timezone: Timezone;
 }
 
-export function timeFormat(scale: AxisScale): Unary<Date, string> {
+type TimeFormatter = Unary<moment.Moment, string>;
+
+const formatterFromDef = (format: string): TimeFormatter => time => time.format(format);
+
+export function pickTimeFormatter(scale: AxisScale): TimeFormatter {
   const ticks = scale.ticks();
-  if (ticks.length < 2) return d3.time.format("%c");
+  if (ticks.length < 2) return formatterFromDef("YYYY-MM-DD HH:mm");
   const first = ticks[0];
   const last = ticks[ticks.length - 1];
   if (first.getFullYear() !== last.getFullYear()) {
-    return d3.time.format("%Y-%m-%d");
+    return formatterFromDef("YYYY-MM-DD");
```

Comment:
Move format as constant to time.ts, it shows inconsistencies.

Reply:
But we format labels differently in different parts of UI. Unification of formats should be a different task.

Reply:
Delegate to #288